cmd/sidecars/network-passt-binding/server/server.go

```diff
@@ -84,13 +91,19 @@ func (s V1alpha2Server) OnDefineDomain(_ context.Context, params *hooksV1alpha2.
 		return nil, err
 	}
 
-	return &hooksV1alpha2.OnDefineDomainResult{
+	return &hooksV1alpha3.OnDefineDomainResult{
 		DomainXML: newDomainXML,
 	}, nil
 }
 
-func (s V1alpha2Server) PreCloudInitIso(_ context.Context, params *hooksV1alpha2.PreCloudInitIsoParams) (*hooksV1alpha2.PreCloudInitIsoResult, error) {
-	return &hooksV1alpha2.PreCloudInitIsoResult{
+func (s V1alpha3Server) PreCloudInitIso(_ context.Context, params *hooksV1alpha3.PreCloudInitIsoParams) (*hooksV1alpha3.PreCloudInitIsoResult, error) {
+	return &hooksV1alpha3.PreCloudInitIsoResult{
 		CloudInitData: params.GetCloudInitData(),
 	}, nil
 }
+
+func (s V1alpha3Server) Shutdown(_ context.Context, _ *hooksV1alpha3.ShutdownParams) (*hooksV1alpha3.ShutdownResult, error) {
+	log.Log.Info("Hook's Shutdown callback method has been called")
```

Comment:
Can you please rephrase it? It does not look good to call it "I was called".
Perhaps "Shutdown passt network binding".